packages/cli/src/commands/doctor/healthchecks/iosDeploy.ts

```diff
@@ -1,58 +1,116 @@
 import execa from 'execa';
+import chalk from 'chalk';
+// @ts-ignore untyped
+import inquirer from 'inquirer';
 import {checkSoftwareInstalled, PACKAGE_MANAGERS} from '../checkInstallation';
 import {packageManager} from './packageManagers';
-import {logManualInstallation} from './common';
+import {logManualInstallation, removeMessage} from './common';
 import {HealthCheckInterface} from '../types';
+import {Ora} from 'ora';
 
-const getInstallationCommand = () => {
+const label = 'ios-deploy';
+
+const installationWithYarn = 'yarn global add ios-deploy';
+const installationWithNpm = 'npm install ios-deploy --global';
+
+const identifyInstallationCommand = () => {
   if (packageManager === PACKAGE_MANAGERS.YARN) {
-    return 'yarn global add ios-deploy';
+    return installationWithYarn;
   }
 
   if (packageManager === PACKAGE_MANAGERS.NPM) {
-    return 'npm install ios-deploy --global';
+    return installationWithNpm;
   }
 
   return undefined;
 };
 
+const installLibrary = async ({
+  installationCommand,
+  packageManagerToUse,
+  loader,
+}: {
+  installationCommand: string;
+  packageManagerToUse: 'yarn' | 'npm';
+  loader: Ora;
+}) => {
+  try {
+    loader.start(`${label} (installing with ${packageManagerToUse})`);
+
+    const installationCommandArgs = installationCommand.split(' ');
+
+    await execa(installationCommandArgs[0], installationCommandArgs.splice(1));
+
+    loader.succeed(`${label} (installed with ${packageManagerToUse})`);
+  } catch (_error) {
```

Comment:
I think we should reject with some kind of error, since this function is returned